ontology/yaml/resources/HVAC/entity_types/MAU.yaml

```diff
@@ -668,6 +668,19 @@ MAU_DFSS_DFVSC_HWDDC_DX2DDC_ZTM_DFM:
   - ZTM
   - DFM
 
+MAU_SFSS_SFVSC_HWSC_CHWSC_RSPC_FDPM2X:
+  description: "Makeup air unit with heating, cooling and return static pressure control."
+  is_canonical: true
+  implements:
+  - MAU
+  - SFSS
+  - SFVSC
+  - HWSC
+  - CHWSC
+  - RSPC
```

Comment:
By definition, MAUs do not have return air (they only provide conditioned outside air to the building).

Reply:
The correction is done. It was an error.